src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -81,6 +81,10 @@ public static boolean hasNoPickupFlag(@Nonnull Item item) {
      */
     public static void markAsNoPickup(@Nonnull Item item, @Nonnull String context) {
         item.setMetadata(NO_PICKUP_METADATA, new FixedMetadataValue(SlimefunPlugin.instance(), context));
+        // Max the pickup delay - This makes it so no Player can pick up items ever without need for an event
+        // It is also one indication used by third party plugins to know if it's a custom item.
+        // Fixes #3203
```

Comment:
*sigh* you guys are a pain
```suggestion
        /*
         * Max the pickup delay - This makes it so no Player can pick up items ever without need for an event.
         * It is also an indication used by third-party plugins to know if it's a custom item.
         * Fixes #3203
         */
```